### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by FractalGeodes**
Since there was pretty much no activity on the repository I went ahead and made some :catstare:

Changes:
- Removed the obligatory quilt template mod stuff from the readme
- `com.example` -> `io.github.sweetberrycollective`
- `example_mod` -> `sweet_crops` ect.

I didn't touch:
- Icon, other than moving it to the correct namespace
- The default mixin or `example_mod.mixins.json`
- Homepage
- Contributors

We'll hold off doing anything else because we're already scared we messed something up somewhere :sweatstare:

### CODE REVIEW

File: src/main/java/com/example/example_mod/mixin/TitleScreenMixin.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.example.example_mod.mixin;
```
Comment (Ampflower) [inline src/main/java/com/example/example_mod/mixin/TitleScreenMixin.java]: Missed repackage, you may want to update this (or just remove this.)

**Review by Ampflower — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/example_mod/mixin/TitleScreenMixin.java (hunk 1 above)
Comment (Ampflower) [reply src/main/java/com/example/example_mod/mixin/TitleScreenMixin.java]: Nevermind, I see that was intentional :foxpple:

Comment (FractalGeodes) [reply src/main/java/com/example/example_mod/mixin/TitleScreenMixin.java]: [ Yeah I'm probably gonna go and remove the title screen mixin this afternoon idk why we kept it ]

**Review by Ampflower — APPROVED** [review]
With noted exceptions, looks good to me so far.

Will ignore the mixin being at the wrong package for now; ideally that particular one would be removed, but in general should be packaged under `io.github.sweetberrycollective.mixin`.

**Comment by FractalGeodes** [comment]
[ Yeah I can't exactly remember why we didn't touch the mixin, I'll remove it this afternoon ]

**Pull request merged**